modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/AdminApiTestHelper.java

```diff
@@ -0,0 +1,158 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package org.wso2.am.integration.test.impl;
+
+import org.testng.Assert;
+import org.wso2.am.integration.clients.admin.api.dto.AdvancedThrottlePolicyDTO;
+import org.wso2.am.integration.clients.admin.api.dto.ApplicationThrottlePolicyDTO;
+import org.wso2.am.integration.clients.admin.api.dto.CustomRuleDTO;
+import org.wso2.am.integration.clients.admin.api.dto.LabelDTO;
+import org.wso2.am.integration.clients.admin.api.dto.SubscriptionThrottlePolicyDTO;
+
+/**
+ * A collection of helper methods to aid Admin REST API related tests
+ */
+public class AdminApiTestHelper {
```

Comment:
Shall we move this in to a helper package instead of keeping it inside the impl

Reply:
Fixed with https://github.com/wso2/product-apim/pull/8750/commits/7aedd50f6ef582bc3d27735b3183e7e955bcb7d3